frontend/src/scenes/billing/Billing.tsx

```diff
@@ -152,13 +159,13 @@ export function Billing(): JSX.Element {
                 </LemonBanner>
             ) : null}
 
-            {!billing?.has_active_subscription && !billing?.trial && platformAndSupportProduct && (
+            {!isManagedAccount && !billing?.has_active_subscription && !billing?.trial && platformAndSupportProduct && (
                 <div className="mb-4">
                     <BillingCTAHero product={platformAndSupportProduct} />
                 </div>
             )}
 
-            <CreditCTAHero />
+            {!isManagedAccount && <CreditCTAHero />}
```

Comment:
Same as other comment, we can do this in billing

Reply:
Removed